Gemfile

```diff
@@ -22,9 +22,11 @@ group :test do
   gem "govuk_schemas"
   gem "minitest-spec-rails"
   gem "mocha"
+  gem "parser", require: false
   gem "rails-controller-testing"
   gem "simplecov"
   gem "simplecov-rcov"
+  gem "unparser", require: false
```

Comment:
We only actually need this one, as it depends on `parser`. These also seem to have leaked into this commit from the one where we add the converter. If they stay here, could we explain why they're being added in the commit message? This would help someone easily identify them for removal in future.